src/test/data/JsonUserPrefsStorageTest/TypicalUserPref.json

```diff
@@ -5,7 +5,8 @@
     "windowCoordinates" : {
       "x" : 300,
       "y" : 100
-    }
+    },
+    "isMaximized" : false
```

Comment:
Should this be updated with the divider position?

Reply:
Yes, it has been updated